qemu/tests/drive_mirror_cancel.py

```diff
@@ -24,7 +24,7 @@ def run_drive_mirror_cancel(test, params, env):
         bg = utils.InterruptedThread(mirror_test.cancel,)
         bg.start()
         job = mirror_test.get_status()
-        if job["type"] != "mirror":
+        if job.get("type", "0") != "mirror":
```

Comment:
@xutian , you are right, it will prompt error since "type" wont exist in the dict when job canceled immediately, but I want it execute below line "raise error.TestFail("Job cancel immediacatly")", so that it is a FAIL instead of an ERROR.